docs/docs/features/plugins.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: default
+title: Plugins
+parent: Features
+nav_order: 9
+---
+
+# Plugins
+{: .no_toc }
+
+* toc
+{:toc}
+
+## Prometheus plugin
+
+### Overview
+This plugin gathers BlazingMQ broker statistic and sends it to [Prometheus](https://prometheus.io/) monitoring system. Both `push` and `pool` modes of interaction with Prometheus are supported.
+
+### Configuration
+By default, plugin is disabled. To enable and configure it, edit `bmqbrkcfg.json` file as follows:
+
+1. Enable plugin and provide path to plugin library
+   ```json
+   "appConfig": {
+       ...
+       "plugins": {
+           "libraries": ["<path-to-prometheus-plugin-library-folder>"],
+           "enabled": ["PrometheusStatConsumer"]
+       }
+   }
+   ```
+
+2. Provide plugin configuration
+   ```json
+   "appConfig": {
+       "stats": {
+           "snapshotInterval": 1,
```

Comment:
Does this value correspond to the [`publishInterval()`](https://github.com/bloomberg/blazingmq/blob/2e158c6da196e2845ece26fdb0afd21ba97bf7f7/src/groups/mqb/mqbplug/mqbplug_statconsumer.h#L88) value from `mqbplug::StatConsumer`? If so, it should be named as such.

Reply:
Both parameters are described [below](https://github.com/bloomberg/blazingmq/pull/261/files#diff-33465695957b579208e81137ec192d48c8e6ff746f48a36fd7a7d96179d64908R60-R61). In a few words, `snapshotInterval` is the interval betwen two secuential updates of metrics by the **broker**. `publishInterval` is the interval between sending metrics to a monitoring system by the **plugin**. I've added some information below